java-checks/src/main/java/org/sonar/java/filters/SuppressWarningFilter.java

```diff
@@ -60,8 +61,20 @@
       .put("overrides", Collections.singleton("java:S1206"))
       .put("removal", Collections.singleton("java:S5738"))
       .put("serial", Collections.singleton("java:S2057"))
-      .put("static", Collections.singleton("java:S2209"))
+      .put("static", SetUtils.immutableSetOf("java:S2696", "java:S2209"))
       .put("rawtypes", Collections.singleton("java:S3740"))
+      // Eclipse (IDE) warnings
+      .put("boxing", SetUtils.immutableSetOf("java:S2153", "java:S5411"))
+      .put("hiding", Collections.singleton("java:S4977"))
+      .put("javadoc", Collections.singleton("java:S1176"))
+      .put("null", Collections.singleton("java:S2259"))
+      .put("resource", SetUtils.immutableSetOf("java:S2093", "java:S2095"))
+      .put("serial", Collections.singleton("java:S2057"))
+      .put("static-access", SetUtils.immutableSetOf("java:S2696", "java:S2209"))
+      .put("static-method", Collections.singleton("java:S2325"))
+      .put("sync-override", Collections.singleton("java:S3551"))
+      .put("unused", SetUtils.immutableSetOf("java:S1481","java:S1065","java:S1854","java:S1068"
```

Comment:
Just for consistency, it's better to keep spaces after comas and keep comas  at the end of the previous line.